test/format/t.c

```diff
@@ -104,7 +104,6 @@ main(int argc, char *argv[])
 		default:
 			usage();
 		}
-	argc -= __wt_optind;
```

Comment:
Was this supposed to be the removal of the next line here.

Reply:
> Was this supposed to be the removal of the next line here.

Please check me to be sure, but I think this is correct, that is, `argc` is not subsequently used and so this assignment is unnecessary, `argv` is subsequently used and its assignment should remain.